app/src/main/kotlin/com/twain/interprep/data/repository/ResourceRepositoryImpl.kt

```diff
@@ -27,11 +27,14 @@ class ResourceRepositoryImpl(private val resourceDAO: ResourceDAO) : ResourceRep
 
     @WorkerThread
     override suspend fun updateResource(resource: Resource) {
-      resourceDAO.updateResource(resource)
+        resourceDAO.updateResource(resource)
     }
 
     override suspend fun deleteResource(resource: Resource) {
         resourceDAO.deleteResource(resource)
     }
+    @WorkerThread
+    override suspend fun getResourceWithLinksBySearchText(searchText: String) =
+        resourceDAO.getResourceWithLinksBySearchText("%$searchText%")
```

Comment:
The wild card is added before and after the string to match the text in the Resource table and ResourceLink table

Reply:
There is one implementation decision: Currently, all the search is case sensitive, we could change this decision.

Reply:
We should definitely support case-insensitive search.

Reply:
By the search is not case sensitive, I mean the search is case-insensitive, so I guess that I implement it correctly

Reply:
Oh then we are good.